lib/matplotlib/tests/test_figure.py

```diff
@@ -216,6 +217,28 @@ def test_figaspect():
     assert h / w == 1
 
 
+@cleanup
+def test_stack_remove():
+    '''
+    Before fixing the bug, doing the following will give you following error:
```

Comment:
This docstring is not very useful. The reason is that nose uses the first line as the name of the test. So the test gets called "Before fixing the bug, doing the following will give you following error:" which is just wrong.

Anyway, I don't think this docstring provides any useful information, because it is clear from the name and the test code itself what it is. I think you can just drop the docstring entirely.